### PR TITLE
fix(Slider): Change align `start` to `flex-start`

### PR DESCRIPTION
**Pull request by gretanausedaite**
Solves #824

### CODE REVIEW

**Review by FlyersPh9 — APPROVED** [review]

File: packages/itwinui-css/backstop/tests/non-ideal-state.html [hunk 1]
```diff
@@ -30,7 +30,7 @@
 
       .demo-non-ideal-state-scrolling {
         height: 100vh;
-        scroll-snap-align: start;
+        scroll-snap-align: flex-start;
```
Comment (rohan-kadkol) [inline packages/itwinui-css/backstop/tests/non-ideal-state.html:33]: flex-start didn't work here when I tested it. Maybe can replace with `center` ([Docs](https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-snap-align#syntax)).
```suggestion
        scroll-snap-align: center;
```
Also, to confirm, the warning in [#824](https://github.com/iTwin/iTwinUI/issues/824) is for slider.css, so maybe this change is unneeded.

Comment (gretanausedaite) [reply packages/itwinui-css/backstop/tests/non-ideal-state.html:33]: I can undo this change.

**Comment by mayank99** [comment]
This PR will only get rid of the warning in one component. Have you gone through all instances of align-items: start​ and justify-content: start​ and even end​ in all components?
________________________________

**Comment by rohan-kadkol** [comment]
> I could update align/justify `end`.

Not sure, but I think `start` and `end` are recommended to be replaced with `flex-start` and `flex-end` respectively, when using `display: flex`.

But for `scroll-snap-align`, I think `start`/`end` only work, and so don't need any replacements. So I don't think the mentioned warning was for `scroll-snap-align`.

**Comment by mayank99** [comment]
> > I could update align/justify `end`.
> 
> Not sure, but I think `start` and `end` are recommended to be replaced with `flex-start` and `flex-end` respectively, when using `display: flex`.
> 
> But for `scroll-snap-align`, I think `start`/`end` only work, and so don't need any replacements. So I don't think the mentioned warning was for `scroll-snap-align`.

Yes, this is correct. Safari does not support `start` and `end` values for `justify-` and `align-` properties when using `display: flex` only (grid works fine). I went through all instances and didn't find any more that need to be changed (either they were already using `flex-start`/`flex-end` or they were using `end` with grid).

**Review by mayank99 — APPROVED** [review]

**Review by rohan-kadkol — APPROVED** [review]

**Pull request merged**